src/category_theory/limits/shapes/biproducts.lean

```diff
@@ -314,4 +317,130 @@ instance biprod.snd_epi {X Y : C} [has_binary_biproduct.{v} X Y] :
 -- If someone is interested, they could provide the constructions:
 --   has_binary_biproducts ↔ has_finite_biproducts
 
+section preadditive
+variables [preadditive.{v} C]
+
+open category_theory.preadditive
+
+/-- A preadditive binary biproduct is a bicone on two objects `X` and `Y` satisfying a set of five
+    axioms expressing the properties of a biproduct in additive terms. The notion of preadditive
+    binary biproduct is strictly stronger than the notion of binary biproduct (but it can be shown
```

Comment:
"but it can be show": has this been done in mathlib? If so, please add a pointer to the declaration.

Reply:
Yes, the necessary construction is part of this PR. I have added this to the docstring.